hazelcast/src/test/java/com/hazelcast/config/MapConfigTest.java

```diff
@@ -1,238 +1 @@
-/*
```

Comment:
I didn't actually - I mean that I didn't delete anything but I'll try to figure out - what's happened exactly.